localisation/00_zuwc_united_localisation.csv

```diff
@@ -2665,85 +2662,85 @@ EVTDESC_WCWAR_708_murloc;They appeared from beneath the waves. In a matter of ho
+spawn_small_qiraji_troops_tooltip;A �YQiraji�! regiment arrives in �Y[target_province.GetName]�!;;Ein �YQiraji�!-Regiment erscheint in �Y[target_province.GetName]�!;;;;;;;;;;;x
+spawn_small_legion_troops_tooltip;A �YLegion�! regiment arrives in �Y[target_summoning_location.GetName]�!;;Ein �YLegions�!-Regiment erscheint in �Y[target_summoning_location.GetName]�!;;;;;;;;;;;x
+spawn_small_horde_troops_tooltip;A �YHorde�! regiment arrives from �YDraenor�!;;Ein �YHorden�!-Regiment erscheint aus �YDraenor�!;;;;;;;;;;;x
+spawn_small_scourge_troops_tooltip;A �YScourge�! regiment arrives in �Y[target_province.GetName]�!;;Ein �YGei�el�!-Regiment erscheint in �Y[target_province.GetName]�!;;;;;;;;;;;x
+EVTTITLE_WCORD_406;The Kor'kron Elite;;Die Kor'kron-Elite;;;;;;;;;;;x
+EVTDESC_WCORD_406;[target_warchief.GetBestName] has announced the start of a recruitment drive intended to assemble the Horde's greatest warriors for a new elite guard commanded by the Warchief: the Kor'Kron. This unit of brave and ferocious soldiers is supposedly dedicated to protecting the very mission statement of the Horde and what it stands for, though it comes with a small caveat: it is directly subordinate to the Warchief, who shall personally handpick [target_warchief.GetHerHis] guard from almost exclusively [target_warchief.GetHerHis] fellow orcs. This has not gone unnoticed by the rest of the Horde's members. They are already calling the Kor'Kron the "Warchief's Praetorian" or "personal guards", and there is a general consensus that it is unlikely that the Kor'kron will defend the common interests and beliefs of the Horde's other members.;;[target_warchief.GetBestName] hat den Beginn einer Anwerbungskampagne angek�ndigt, mit der die gr��ten Krieger der Horde f�r eine neue Elitegarde unter dem Kommando des Kriegsh�uptlings zusammengestellt werden sollen: die Kor'Kron. Diese Einheit aus mutigen und wilden Soldaten soll dem Schutz des eigentlichen Leitbildes der Horde und dessen, wof�r sie steht, dienen, allerdings mit einem kleinen Vorbehalt: Sie ist direkt dem Kriegsh�uptling unterstellt, [target_warchief.Getderdie] pers�nlich ihre Mitglieder fast ausschlie�lich aus [target_warchief.GetHerHis]en Mit-Orks ausw�hlt, was auch den anderen Mitgliedern der Horde nicht verborgen geblieben ist. Sie bezeichnen die Kor'Kron bereits als �Pr�torianer des Kriegsh�uptlings� oder �pers�nliche Wachen�, und es besteht ein allgemeiner Konsens dar�ber, dass es unwahrscheinlich ist, dass die Kor'kron die gemeinsamen Interessen und �berzeugungen der anderen Mitglieder der Horde verteidigen werden.;;;;;;;;;;;x
```

Comment:
@DukeJeans take a look at changes here. They look better than the current version?